src/markdown/elements.rs

```diff
@@ -247,6 +271,14 @@ impl CodeLanguage {
     pub(crate) fn supports_auto_render(&self) -> bool {
         matches!(self, Self::Latex | Self::Typst)
     }
+
+    pub(crate) fn hidden_line_prefix(&self) -> Option<&'static str> {
+        match self {
+            CodeLanguage::Rust => Some("# "),
+            CodeLanguage::Python | CodeLanguage::Shell(_) | CodeLanguage::Bash => Some("## "),
```

Comment:
I wonder if there is a better prefix over to use for these languages since a single `#` denotes a comment in them, so if you wanted to show comments in the snippet, that could potentially be confusing?

Reply:
How about using `///`? I only dislike that in rust because it has a meaning but for the rest of the languages that could work.

Reply:
That makes sense, can roll with `///`.